Gemfile

```
@@ -15,7 +15,7 @@ gem "govuk_publishing_components", "~> 21.34.0"
 gem "htmlentities", "~> 4.3"
 gem "jbuilder", "~> 2.5"
 gem "jquery-rails"
-gem "kaminari", "~> 1.2.1"
+gem "kaminari", "= 1.1.1"
```

```suggestion
gem "kaminari", "< 1.2"
```
This will allow patches and bugfixes (e.g. version 1.1.2)

---

ok sure